doc/arkode/guide/source/Usage/ERKStep_c_interface/User_callable.rst

```diff
@@ -992,6 +992,8 @@ Optional inputs for IVP method selection
    +----------------------------------+---------------------------------+------------------+
    | Specify explicit RK table number | :c:func:`ERKStepSetTableNum()`  | internal         |
    +----------------------------------+---------------------------------+------------------+
+   | Specify explicit RK table name   | :c:func:`ERKStepSetTableName()` | internal         |
```

Comment:
Same comment as in the ARK section. Prefer "Set explicit RK table via its name" (and fix line above similarly).